homeassistant/components/qbittorrent/sensor.py

```diff
@@ -109,16 +119,17 @@ class QBittorrentSensor(SensorEntity):
     def __init__(
         self,
         description: SensorEntityDescription,
-        qbittorrent_client,
-        client_name,
+        qbittorrent_client: Client,
+        config_entry: ConfigEntry,
         exception,
```

Comment:
Maybe for a preliminary PR:
What is the point of this argument? Why not simply use `except LoginRequired` on line 133?

Reply:
Great idea, not sure why the original implementation includes this argument.

https://github.com/home-assistant/core/pull/90411